roles/k8s-ut/tasks/main.yml

```diff
@@ -1,13 +1,11 @@
 - name: Install prereq packages
-  yum:
-    name: "{{ packages }}"
-    state: present
-  vars:
-    packages:
+  package:
+    name:
       - git
       - make
       - gcc
-  when: ansible_pkg_mgr == 'yum'
+    state: present
+  when: ansible_pkg_mgr == ['yum','dnf']
```

Comment:
so this also needs to be `in` instead of `==`?

Reply:
Oops. I have addressed it.